lib/followups/followup_sender.rb

```diff
@@ -6,13 +6,11 @@ class FollowupSender
     def self.send_messages
       users = WifiUser::User
         .where(last_login: nil)
-        .where { created_at <= (Date.today - DAYS_BEFORE_SENDING) }
+        .where { created_at <= (Time.now - 24 * 3600 * DAYS_BEFORE_SENDING) }
```

Comment:
could "24 * 3600" be a constant eg, SECONDS_IN_DAY or something?  Just to make the code easier to understand / read?

Reply:
👍